PythonWrapperGenerator/PythonWrapperGenerator.pro

```diff
@@ -0,0 +1,62 @@
+TARGET = pythonwrappergen
+
+TEMPLATE = app
+CONFIG += console
+CONFIG -= app_bundle
+
```

Comment:
I mentioned this in a previous commit, but I'm not sure if it will be shown properly, so here it is again:
* we should use a precompiled header with all the stl+clang libraries